gcp-observability/src/main/java/io/grpc/gcp/observability/ObservabilityConfigImpl.java

```diff
@@ -100,19 +102,23 @@ private void parseConfig(Map<String, ?> config) {
         }
         this.eventTypes = eventTypesBuilder.build();
       }
-      String sampler = JsonUtil.getString(config, "global_trace_sampler");
       Double samplingRate = JsonUtil.getNumberAsDouble(config, "global_trace_sampling_rate");
-      checkArgument(
-          sampler == null || samplingRate == null,
-          "only one of 'global_trace_sampler' or 'global_trace_sampling_rate' can be specified");
-      if (sampler != null) {
-        this.sampler = new Sampler(SamplerType.valueOf(sampler.toUpperCase()));
+      if (enableCloudTracing && samplingRate == null) {
```

Comment:
Would it simpler/better if you just remove the check for `enableCloudTracing` and do it as follows (with epsilon defined as a class constant)?:

```
      if (samplingRate == null) {
        this.sampler = Samplers.probabilitySampler(0.0);
      } else {
        checkArgument(
            samplingRate >= 0.0 && samplingRate <= 1.0,
            "'global_trace_sampling_rate' needs to be between [0.0, 1.0]");
        // Using alwaysSample() instead of probabilitySampler() because according to
        // {@link io.opencensus.trace.samplers.ProbabilitySampler#shouldSample}
        // there is a (very) small chance of *not* sampling if probability = 1.00.
        if (Math.abs(1 - samplingRate) < epsilon) {
          this.sampler = Samplers.alwaysSample();
        } else {
          this.sampler = Samplers.probabilitySampler(samplingRate);
        }
      }
```

Reply:
Done